src/itemEditableInfo/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+
+import { TextDisplayType } from 'text'
+import Item from '_utils/item'
+import { A11yProps, pickA11yProps } from '_utils/interfaces'
+import { color } from '_utils/branding'
+
+export interface ItemEditableInfoProps extends A11yProps {
```

Comment:
Are we keeping interfaces instead of types?

Reply:
I need the old a11yProps pick mechanism :/

Reply:
This should work:

```ts
export type ItemEditableInfoProps = A11yProps & {
  // ...
}
```

Reply:
I updated it to the above, it works ! Thanks folks !